vcfconcat.c

```diff
@@ -549,6 +553,114 @@ static void concat(args_t *args)
     }
 }
 
+BGZF *hts_get_bgzfp(htsFile *fp);
+static void naive_concat(args_t *args)
+{
+    // only compressed BCF atm
+
+    BGZF *bgzf_out;
+    if ( strcmp("-",args->output_fname) )
+        bgzf_out = bgzf_open(args->output_fname,"w");
+    else
+        bgzf_out = bgzf_dopen(fileno(stdout), "w");
+
+    int page_size = getpagesize();
+    char *buf = (char*) valloc(page_size);
```

Comment:
The harm in using obscure and deprecated functions is code portability.  IIRC on Windows I couldn't use sysconf either in the Staden Package as it simply didn't exist on MSC.  Maybe it does now, but it demonstrates the point still.  Hard coded numbers are obscure, but sufficient if commented AND PORTABLE!  

Besides, the whole argument over valloc vs malloc is kind of irrelevant when it could just be ```char buf[BUF_SIZE]``` with a define.   (I'm happy with either malloc or fixed buffer btw, but not with valloc.)

There's also nothing wrong in using a fixed buffer size.  If you _really_ care over what it should be, then the right thing to do is to stat the file and look at the filesystem preferred block size ```st_blksize```.

Anyway, as it currently stands is sufficient and we need to get the release out of the door!